lib/matplotlib/quiver.py

```diff
@@ -201,6 +201,20 @@
 See Also
 --------
 .Axes.quiverkey : Add a key to a quiver plot.
+
+Notes
+-----
+The arrow is drawn as a polygon using the nodes as shown below. The values
+*headwidth*, *headlength* and *headaxislength* are in units of width.
+
+.. image:: /static/quiver_sizes.svg
```

Comment:
```suggestion
.. image:: /_static/quiver_sizes.svg
```
(as I am sure you will realize yourself...)